src/physics/utils/tropopause_read_file.meta

```diff
@@ -0,0 +1,29 @@
+[ccpp-table-properties]
+  name = tropopause_read_file
+  type = module
+
+[ccpp-arg-table]
+  name  = tropopause_read_file
+  type  = module
+[ tropp_slices ]
+  standard_name = number_of_months_in_year
```

Comment:
Thanks for the suggestion. I'm a bit on the fence about this here. Since we changed everything in `tropopause_climo_file` to use `tropp_slices = 12` as dimensions I'm concerned that a future change (however unlikely) of the number of time slices used in the tropopause climatology file would need to separate this back into `tropopause_climo_file` if we lift this into `registry.xml` today.

Especially if there's this hardcoded in `tropopause_climo_file`...
```fortran
    ! YMD (01-16, 02-14, ...) corresponding to the time slices of the climatological
    ! tropopause dataset. Will be converted to day-of-year and stored in tropp_days.
    integer, parameter :: dates(tropp_slices) = (/ 116, 214, 316, 415, 516, 615, &
         716, 816, 915, 1016, 1115, 1216 /)
```

To avoid the hidden effect of `number_of_months_in_year` controlling `tropp_slices` here I wonder if we could just rename `tropp_slices`'s standard name to `number_of_time_slices_in_climatology_dataset`. I checked and such a change would not affect `atmospheric_physics`.

Sorry if it's too much thought into a simple variable that might never change! It just makes me slightly uncomfortable that there's some kind of hidden side effect that will be certainly forgotten months later that `number_of_months_in_year` is hardcoded into the # of tropopause slices.

Reply:
Opened an issue for this since the atmos_phys new directory structure has to be brought in first

https://github.com/ESCOMP/CAM-SIMA/issues/308